README.md

```diff
@@ -188,6 +188,7 @@ import chartmogul
 chartmogul.Invoice.create(config, uuid='cus_5915ee5a-babd-406b-b8ce-d207133fb4cb', data={})
 chartmogul.Invoice.all(config, uuid='cus_5915ee5a-babd-406b-b8ce-d207133fb4cb', page=2, per_page=10)
 chartmogul.Invoice.all(config, customer_uuid='cus_f466e33d-ff2b-4a11-8f85-417eb02157a7', external_id='INV0001')
+chartmogul.Invoice.retrieve(config, uuid='inv_123')
```

Comment:
If you're using a UUID here, then display a valid UUID, or it will confuse people.